bitcoin/src/lib.rs

```diff
@@ -59,9 +59,9 @@ pub extern crate base64;
 /// Encodes and decodes the Bech32 forrmat.
 pub extern crate bech32;
 
-#[cfg(feature = "bitcoinconsensus")]
 /// Bitcoin's libbitcoinconsensus with Rust binding.
-pub extern crate bitcoinconsensus;
+#[cfg(feature = "bitcoinconsensus")]
+extern crate bitcoinconsensus;
```

Comment:
I don't think this is needed at all now.

Reply:
I had the same thought, but didn't check. (In Rust post-2018 non-`pub` `extern crate`s are no-ops, I believe.)

Reply:
`pub` are not no-ops (they are reexports) but non-pub are.